internal/wire/analyze.go

```diff
@@ -207,6 +212,28 @@ dfs:
 				valueExpr:     v.expr,
 				valueTypeInfo: v.info,
 			})
+		case pv.IsField():
+			f := pv.Field()
+			if index.At(f.Parent) == nil {
+				// Make sure to visit the parent struct first. This is likely a
+				// Value call.
```

Comment:
I don't understand this comment: why is likely to be a `Value` call?

Reply:
But it's not due to being a given or not: it's because the DFS hasn't traversed the node yet.